src/runtime/storageNG/tests/store-test.ts

```diff
@@ -103,6 +103,7 @@ describe('Store', async () => {
 
     let sentSyncRequest = false;
 
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
remove this line, remove `async` from promise executor and remove `await` before `activeStore.onProxyMessage`

Reply:
done